app/controllers/admin_public_body_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
       end
       @info_requests = info_requests.paginate(:page => params[:page],
                                               :per_page => 100)
+      @versions = @public_body.versions.order('version desc')
```

Comment:
Super picky, but for clarity, we should probably either use the hash syntax `order(version: :desc)` or capitalisise `DESC` so that it's clear this is SQL.